<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GIMR Video</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="video1">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="video2">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>886</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Connections</string>
    </property>
    <widget class="QMenu" name="menuCamera_1">
     <property name="title">
      <string>Camera 1</string>
     </property>
     <addaction name="actionConnection"/>
    </widget>
    <widget class="QMenu" name="menuCamera_2">
     <property name="title">
      <string>Camera 2</string>
     </property>
     <addaction name="actionConnection_2"/>
    </widget>
    <addaction name="actionStart_Video"/>
    <addaction name="actionStop_Pause"/>
    <addaction name="separator"/>
    <addaction name="menuCamera_1"/>
    <addaction name="menuCamera_2"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuRun"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionStart_Video">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionStop_Pause">
   <property name="text">
    <string>Stop/Pause</string>
   </property>
  </action>
  <action name="actionConnection">
   <property name="text">
    <string>Connection</string>
   </property>
  </action>
  <action name="actionConnection_2">
   <property name="text">
    <string>Connection</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
